app/src/main/res/drawable/ic_baseline_food_bank_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
```

Comment:
This is a random width, I liked it so I made it 24. but 24 is the universal number for love. and that's why i used it.